lib/src/_shared/DateTextField.jsx

```diff
@@ -211,7 +211,7 @@ export class DateTextField extends PureComponent {
       return;
     }
 
-    e.target.blur();
```

Comment:
why this was removed?

Reply:
Ohh, I’ve tried to test why pickers hangs on inside form control and found that this line is useless. Any thoughts?

Reply:
It doesn't really make any sense to blur input on focus :/
I'm even surprised, that it didn't cause any bugs.
But looks like it can be removed now